### ISSUE TITLE
comma, period etc outside Devanagari scope

### ISSUE THREAD

**Issue by drdhaval2785** [sanskrit-lexicon/csl-corrections#75]
Reference - https://github.com/sanskrit-lexicon/csl-orig/issues/525

There are many dictionaries where they are placed inside Devanagari markup. Need to makd corrections just like the one made in AP90.

This is a placeholder to track progress across dictionaries.

**Comment by Andhrabharati** [comment]
do we have this as the next task, across all the dictionaries?

**Comment by drdhaval2785** [comment]
I vote for this, but this would change the structure substantially. We should first integrate whatever changes you have already made in your local file first.
After this issue, the file differences would increase manifold. 
Keep this on hold till your local changes are incorporated @Andhrabharati .

**Comment by Andhrabharati** [comment]
@funderburkjim has initiated this work, starting with MW, after my entry here. He understood the need very clearly and later he continued doing the same in AP90 and then PWG.

I just thought this could be done in all works at one go.

There are two ways of handling corrections-- pointwise across all books, or bookwise for all points in it.

**Comment by drdhaval2785** [comment]
Let's complete European language texts addition to MW first. We will handle this comma, period etc after that.

**Comment by Andhrabharati** [comment]
well taken!

will post my file parts along with cologne lines today. I was exhausted yesterday by the hectic work (Ind. Spr., SKD and then MW) and could do only half of the splitting yesterday.

**Comment by drdhaval2785** [comment]
Terminal commas are outside the Devanagari scope for all dictionaries in CDSL now.
The commas within the text are still not handled.

**Comment by drdhaval2785** [comment]
Periods can not be handled mechanically, as there are some places where the periods represent danda in Devanagari. Removing them outside the Devanagari scope would not be advisable.

**Comment by drdhaval2785** [comment]
Non-terminal comma, semicolon etc may be handled elsewhere. Closing this issue
